<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmtpAdvancedSettingsWidget</class>
 <widget class="QWidget" name="SmtpAdvancedSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>372</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
   <item row="3" column="0">
    <widget class="nx::vms::client::desktop::PasswordInputField" name="passwordInputField" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="portLabel">
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="buddy">
      <cstring>portComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="unsecuredRadioButtonLayout">
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QWidget" name="unsecuredRadioButtonAligner" native="true"/>
     </item>
     <item>
      <widget class="QRadioButton" name="unsecuredRadioButton">
       <property name="text">
        <string>Unsecure Connection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="unsecuredRadioButtonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="nx::vms::client::desktop::InputField" name="signatureInputField" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="portComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="nx::vms::client::desktop::InputField" name="serverInputField" native="true"/>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="tlsRadioButtonLayout">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QWidget" name="tlsRadioButtonAligner" native="true"/>
     </item>
     <item>
      <widget class="QRadioButton" name="tlsRadioButton">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Secure connection using TLS.</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tlsRecommendedLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(recommended)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>tlsRadioButton</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="tlsRadioButtonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="sslRadioButtonLayout">
     <item>
      <widget class="QWidget" name="sslRadioButtonAligner" native="true"/>
     </item>
     <item>
      <widget class="QRadioButton" name="sslRadioButton">
       <property name="text">
        <string>Secure connection using SSL.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sslRecommendedLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>(recommended)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>sslRadioButton</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="sslRadioButtonSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="nx::vms::client::desktop::InputField" name="supportInputField" native="true"/>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="nx::vms::client::desktop::InputField" name="userInputField" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="nx::vms::client::desktop::InputField" name="emailInputField" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::InputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/input_field.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::PasswordInputField</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/password_input_field.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
